conbench/tests/api/_fixtures.py

```diff
@@ -268,6 +268,7 @@ def benchmark_result(
     data["run_id"] = run_id if run_id else _uuid()
     data["batch_id"] = batch_id if batch_id else _uuid()
     data["tags"]["name"] = name if name else _uuid()
+    data["timestamp"] = datetime.now(timezone.utc).replace(microsecond=0).isoformat()
```

Comment:
I think it makes more sense in this function to use `now()` by default as the timestamp, instead of a hardcoded one. This caused most of the test changes in this PR but also fixed the tests that were broken originally, since the new query uses the result timestamp.

Reply:
Yes!!